src/cdk/a11y/input-modality/input-modality-detector.ts

```diff
@@ -100,6 +101,12 @@ export class InputModalityDetector implements OnDestroy {
     return this._modality.value;
   }
 
+  /**
+   * The most recently detected input modality event target. Is null if no input modality has been
+   * detected or if the associated event target is null for some unknown reason.
```

Comment:
I'm not sure in what case the event target would be null, but apparently it's possible (at least per the types).